### PR TITLE
Add an autoconnecting Layertree

### PR DESCRIPTION
**Pull request by marcjansen**
Hat-tip to @chrismayer for the awesome `mapready`-event idea.

### CODE REVIEW

File: app/view/LayerTree.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        if (map) {
+            cfg.store = me.makeLayerStore(map);
+        } else {
+            CpsiMapview.getApplication().on('mapready', me.autoConnectToMap);
+        }
+        me.callParent([cfg]);
+    },
+
+    /**
+     * Gueses the mapcomponent and assigns the appropriate layers store, if one
```
Comment (chrismayer) [inline app/view/LayerTree.js]: Typo `Guesses`?

File: app/view/LayerTree.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/**
+ * A Layertree which autoconnects to the map of the application.
+ */
+Ext.define('CpsiMapview.view.LayerTree', {
+    extend: 'Ext.tree.Panel',
+    xtype: 'layertree',
```
Comment (chrismayer) [inline app/view/LayerTree.js]: Please use the prefix `cmv_` for the xtype.

File: app/view/LayerTree.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+/**
+ * A Layertree which autoconnects to the map of the application.
+ */
+Ext.define('CpsiMapview.view.LayerTree', {
```
Comment (chrismayer) [inline app/view/LayerTree.js]: Should  we place this in a separate folder/package?

Comment (marcjansen) [reply app/view/LayerTree.js]: I wasn't sure. right now we only have on tree…

File: app/view/LayerTree.js (hunk 2 above)
Comment (marcjansen) [reply app/view/LayerTree.js]: Will do.

File: app/view/LayerTree.js (hunk 1 above)
Comment (marcjansen) [reply app/view/LayerTree.js]: Will change, thx

**Pull request merged**